container-storage-setup.sh

```diff
@@ -1814,6 +1841,19 @@ reset_storage() {
     fi
   fi
 
+  # If we created a volume group, remove volume group.
+  if [ "$_M_VG_CREATED" == "1" ];then
+    if ! remove_vg_if_exists "$_M_VG"; then
+      Error "Failed to remove volume group $_M_VG"
+      return 1
+    fi
+
+    # Cleanup any disks we added to volume group.
+    if ! remove_disk_pvs_parts "$_M_DEVS_RESOLVED";then
```

Comment:
No, right now I don't have a separate log file where errors can go. For now I have sent everything to /dev/null.

So where should the errors from lvm commands go. Or should I just redirect stdout to /dev/null and let errors go to stderr?